src/Runner.Worker/ExpressionFunctions/HashFiles.cs

```diff
@@ -0,0 +1,139 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Minimatch;
```

Comment:
Might be able to remove some of these now (e.g. minimatch). Can that be dropped from nuget restore too?

Reply:
the old expression func (part of SDK) still use minimatch, can't drop from nuget now.
I am plan to feature flag the switch in case the new expression func break something.